xray/utils/downloadindexer.go

```diff
@@ -178,3 +182,12 @@ func deleteOldIndexers(indexerDirPath string) error {
 
 	return nil
 }
+
+func getIndexerExtension(os string) string {
```

Comment:
It looks like this functions does not need to receive the OS as an argument. It can figure it out by itself.
Also, I'mm not sure separating the indexer app file name from its extension is the simplest approach. I suggest we simply rename this function to ```getIndexerBinaryname``` and have it return either "indexer-app" or "indexer-app.exe". This allows us to remove both the ```indexerExtension``` variable and also the ```indexerFileName``` constants. Anyone who'll need the idexer app name will simply call this function.

Reply:
Now I see that this is also what @asafgabai suggested above :)